crates/optimism/chainspec/src/lib.rs

```diff
@@ -8,7 +8,7 @@
 #![cfg_attr(all(not(test), feature = "optimism"), warn(unused_crate_dependencies))]
 #![cfg_attr(docsrs, feature(doc_cfg, doc_auto_cfg))]
 // The `optimism` feature must be enabled to use this crate.
-#![cfg(feature = "optimism")]
+//#![cfg(feature = "optimism")]
```

Comment:
we still need this, right?

Reply:
apparently we don't! ci would fail otherwise. we only need this for crates that depend (directly on transitively) on `reth-primitives`

Reply:
(and for crates that depend on other non-optimism crates that still have optimism feature asides `reth-primitives`)

Reply:
but we still keep the feature to skip compiling optimism crates into reth binary right? @mattsse

Reply:
> but we still keep the feature to skip compiling optimism crates into reth binary right?

this crate must not be part of the reth binary because this crate balloons the executable because the genesis files are very large